docs/technical-design/creating-and-testing-endpoints.md

```diff
@@ -10,21 +10,24 @@ The relative merits of each approach should become clear below.
 
 ## GraphQL
 
-The major mental shift in using GraphQL, in contrast to RESTful services, is that you have to _begin_ by thinking of the data you want returned. With REST, it's common to hit an endpoint, get back a large JSON, browse through it, and then, in the application code, whittle it down, or pluck the bits that you need. But we'll construct our GraphQL queries to return exactly what we need, before we hit the application code at all.
+The major mental shift in using GraphQL, in contrast to RESTful services, is that you have to _begin_ by thinking of the data you want returned. With REST, it's common to hit an endpoint, get back a large JSON, browse through it, and then, in the application code, whittle it down, or pluck the bits that you need. But we'll construct our [GraphQL operations (queries and mutations)](https://graphql.org/learn/queries/) to return exactly what we need, before we hit the application code at all.
 
 ### Apollo Studio Explorer
 [Apollo Explorer tool](https://www.apollographql.com/docs/graphos/explorer/) is a web-based IDE for writing and executing GraphQL operations on our deployed GraphQL API, with features such as schema referencing, query linting, autocomplete, and a jump-to-definition tool. With this tool we can test and write GraphQL operations without the need of front-end UI to execute the operation.
 
 We have two ways to access the Apollo Explorer tool:
 
-**Embedded Apollo Explorer Tool**:
+**Embedded GraphQL Explorer Tool**:
```

Comment:
I changed this intentionally because the URL is graphql explorer so I was searching GraphQL Explorer in the codebase trying to find the docs and it wasn't coming up.  Then I started searching " test graphql" and "testing" and wasn't seeing it either. I changed the title of this doc to better suggest it has testing advice as well.